<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamlineDlg</class>
 <widget class="QDialog" name="StreamlineDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Overlay surface streamlines</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>16</number>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QCheckBox" name="cbOverlay">
     <property name="text">
      <string>Show vector field as needle overlay</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Select field</string>
     </property>
     <property name="buddy">
      <cstring>cbSelectField</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QComboBox" name="cbSelectField">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QRadioButton" name="rbAutoScale">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Adapt needle length to local element size</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QRadioButton" name="rbScaleNeedles">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Scaled needle length by magnitude</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QRadioButton" name="rbEqualLength">
     <property name="text">
      <string>Use constant needle length</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="lbScaleFactor">
     <property name="text">
      <string>Needle length</string>
     </property>
     <property name="buddy">
      <cstring>sbNeedleScale</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QScienceSpinBox" name="sbNeedleScale">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <double>-999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999999.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>148</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="pbApply">
     <property name="text">
      <string>&amp;Apply</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="pbClose">
     <property name="text">
      <string>&amp;Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QScienceSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>../qsciencespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbOverlay</tabstop>
  <tabstop>cbSelectField</tabstop>
  <tabstop>rbAutoScale</tabstop>
  <tabstop>rbScaleNeedles</tabstop>
  <tabstop>rbEqualLength</tabstop>
  <tabstop>sbNeedleScale</tabstop>
  <tabstop>pbApply</tabstop>
  <tabstop>pbClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>StreamlineDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbOverlay</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbOverlay</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbSelectField</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbOverlay</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbNeedleScale</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbOverlay</sender>
   <signal>toggled(bool)</signal>
   <receiver>rbAutoScale</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbOverlay</sender>
   <signal>toggled(bool)</signal>
   <receiver>rbScaleNeedles</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbOverlay</sender>
   <signal>toggled(bool)</signal>
   <receiver>rbEqualLength</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
